src/Tizen.Multimedia.Vision/MediaVision/InferenceModelConfiguration.cs

```diff
@@ -332,6 +339,30 @@ public InferenceTargetType Target
             }
         }
 
+        /// <summary>
+        /// Gets or sets the inference model's device.
```

Comment:
Is the description, "Gets or sets the inference model's device.", an appropriate explanation?
How about the following description?

```suggestion
        /// Gets or sets the processor type for inference of deep learning models.
```
If the suggested description has wrong information, please update and then use yours.

Reply:
I'll raise new PR for these requests.
Please review the new PR(https://github.com/Samsung/TizenFX/pull/1580).